### PR TITLE
Table of Installed EVCs and notification for requested circuit response 

### PR DESCRIPTION
**Pull request by Niehaus**
Working on #209 

### :page_facing_up: Release Notes

- [ui-mef_eline] Added Table of Installed EVCs
- [ui-mef_eline] Added notification response of requested circuit

### CODE REVIEW

File: ui/k-info-panel/list_connections.kytos [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          * no EVCs installed.. no need to show the req response */
+        },
+      })
+
+      return tableRows;
+    }
+  },
+  computed: { 
+    emptyTable: function() {
+      return this.list_EVCs().lenght > 0
```
Comment (hdiogenes) [inline ui/k-info-panel/list_connections.kytos:58]: This should be causing errors, length is misspelled.

```suggestion
      return this.list_EVCs().length > 0
```

File: ui/k-toolbar/main.kytos [hunk 2]
```diff
@@ -57,35 +63,62 @@ module.exports = {
...
+        var content = {
+            component: 'kytos-mef_eline-k-info-panel-list_connections',
+            content: {},
+            icon: "desktop",
+            title: "View Connections",
+            subtitle: "by kytos/mef_eline"
+        }
+        this.$kytos.$emit("showInfoPanel", content)
+    },
+    post_succes(data) {
```
Comment (hdiogenes) [inline ui/k-toolbar/main.kytos:76]:
```suggestion
    post_success(data) {
```

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: models.py [hunk 3]
```diff
@@ -590,12 +590,16 @@ def deploy_to_path(self, path=None):
...
+        elif self.uni_a.interface.switch == self.uni_z.interface.switch:
+            self._install_direct_uni_flows()
+            use_path = Path()
+        else:
+            return False
+        self.activate()
+        self.current_path = use_path
+        self.sync()
+        log.info(f"{self} was deployed.")
+        return True
```
Comment (hdiogenes) [inline models.py:602]: Should this code really be here or is it from another branch/PR?

Comment (Niehaus) [reply models.py:602]: I don't think so, should I just drop this commit?

**Review by hdiogenes — APPROVED** [review]
Looks OK to me. @ajoaoff, can you have a look at why the tests are failing here?

**Comment by cmagnobarbosa** [comment]
Related #206

File: ui/k-info-panel/list_connections.kytos [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      <p class='empty-con-list'>No EVCs Installed.<br>
+      Close this info-panel and try again, please.</p>
+    </div>
+
+</template>
+<script>
+module.exports = {
+  data(){
+    return {
+        headers: ['Name ', 'Endpoint A' , 'Tag A' , 'Endpoint Z' , 'Tag z' , 'Enabled' , 'Active'],
```
Comment (hdiogenes) [inline ui/k-info-panel/list_connections.kytos:16]:
```suggestion
        headers: ['Name ', 'Endpoint A' , 'Tag A' , 'Endpoint Z' , 'Tag Z' , 'Enabled' , 'Active'],
```

File: ui/k-info-panel/list_connections.kytos [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              "Enabled": evc.enabled,
+              "Active": evc.active
+            };
+
+            tableRows.push(connection);
+          });
+        },
+        error: function(data) {
+          /* This info-panel will show a message that there's
+          * no EVCs installed.. no need to show the req response */
```
Comment (hdiogenes) [inline ui/k-info-panel/list_connections.kytos:47]:
```suggestion
          /* This info-panel will show a message that there are
           * no EVCs installed - no need to show the response */
```

**Review by hdiogenes — APPROVED** [review]

**Pull request merged**